src/main/java/com/nvkaip/zoom_slack_bot/entity/Meeting.java

```diff
@@ -0,0 +1,139 @@
+package com.nvkaip.zoom_slack_bot.entity;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Meeting {
+
+    private String uuid; //Meeting unique ID.
```

Comment:
```suggestion
    private String uuid; // Meeting unique ID.
```

Reply:
same everywhere

Reply:
Maybe it will be better to create Javadoc comment on the top of the class?